AestraAudio/src/Plugin/PluginScanner.cpp

```
@@ -545,5 +602,41 @@ int PluginScanner::countPluginFiles() const {
     return count;
 }
 
+// ==============================
+// Security: Trusted Paths (SEC-RTM-005)
+// ==============================
+
+bool PluginScanner::isTrustedPath(const std::filesystem::path& path) {
+    std::string p = path.lexically_normal().string();
+
+    // Linux: system-wide paths are trusted
+    if (p.find("/usr/lib/vst3") == 0 ||
```

Cppcheck (2.20.0)

[performance] Inefficient usage of string

(stlIfStrFind)